plugins/woocommerce/src/Blocks/Utils/CartCheckoutUtils.php

```diff
@@ -146,4 +146,80 @@ function( $acc, $zone ) {
 		];
 		return $formatted_shipping_zones;
 	}
+
+	/**
+	 * Syncronises the attributes of the express payment blocks between the Cart and Checkout blocks.
+	 *
+	 * @param int     $post_id The post ID.
+	 * @param WP_Post $post The post object.
+	 * @param string  $cart_or_checkout The block type to check.
+	 * @param array   $default_settings Default settings.
+	 *
+	 * @return array An array of formatted payment gateways.
+	 */
+	public static function update_express_payment_settings( $post_id, $post, $cart_or_checkout, $default_settings ) {
+
+		if ( 'cart' !== $cart_or_checkout && 'checkout' !== $cart_or_checkout ) {
```

Comment:
```suggestion
		if ( ! in_array( $cart_or_checkout, array( 'cart', 'checkout' ), true ) ) {
```

Would this simplify?